### ISSUE TITLE
Nested Lumberjack triggers "newNode is not defined" js error

### ISSUE THREAD

**Issue by cwchong** [silverstripe/silverstripe-lumberjack#81]
SS4.1

Consider a situation where the parent page **P** has "lumberjacked" child pages **C**, which in turn also have "lumberjacked" grandchild pages **G**.
A grandchild page **G** when edited, cannot be rendered in the Sitetree navigation, because its parent **C** node is hidden in the navigation.

I believe that is the cause of the js error.

**Comment by robbieaverill** [comment]
So I guess we always need to show the selected page C when you're editing its children (G), regardless of whether the pages are set to be hidden via lumberjack in the SiteTree - would that make sense?

**Comment by cwchong** [comment]
@robbieaverill yes that would make sense, and to generalise it should be, when editing a selected page G, all its ancestors should be unhidden in the sitetree

**Comment by NightJar** [comment]
Sounds a bit like the search filter funcitonality on the SiteTree structure itself
